### PR TITLE
Change access modifiers of debug config classes for reusing code

### PR DESCRIPTION
**Pull request by musienko-maxim**
### What does this PR do?
* Change access modifiers in some methods for using this one in other tests
* Apply selenium helper to `AbstractDebugConfig` class change depended classes
@dmytro-ndp, @Ohrimenko1988, @SkorikSergey

### CODE REVIEW

**Review by Ohrimenko1988 — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/debug/AbstractDebugConfig.java [hunk 1]
```diff
@@ -91,28 +96,19 @@ public void waitDebugConfigurationIsClosed() {
    */
   public void removeConfig(String configName) {
     String configItemXpath = String.format(Locators.CONFIG_ITEM_XPATH_TEMPLATE, configName);
-    new WebDriverWait(seleniumWebDriver, REDRAW_UI_ELEMENTS_TIMEOUT_SEC)
-        .until(ExpectedConditions.visibilityOfElementLocated(By.xpath(configItemXpath)))
-        .click();
+    seleniumWebDriverHelper.waitAndClick(By.xpath(configItemXpath));
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/debug/AbstractDebugConfig.java]: Empty rows looks redundant here.

**Review by dmytro-ndp — APPROVED** [review]

**Comment by musienko-maxim** [comment]
ci-test

**Review by SkorikSergey — APPROVED** [review]

**Comment by musienko-maxim** [comment]
ci-build

**Comment by musienko-maxim** [comment]
ci-test

**Comment by riuvshin** [comment]
Results of automated E2E tests of Eclipse Che Multiuser on OCP:
[Build details](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1121/)
[Test report](https://ci.codenvycorp.com/job/che-pullrequests-test-ocp/1121//Selenium_20tests_20report/)
docker image: eclipseche/che-server:11785
https://github.com/orgs/eclipse/teams/eclipse-che-qa please check this report.

**Pull request merged**